client/web-sveltekit/src/lib/LanguageIcon.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+    import { mdiFileCodeOutline } from '@mdi/js'
+
+    import { getFileIconInfo } from '$lib/wildcard'
+
+    import Icon from './Icon.svelte'
+
+    export let language: string
+    const icon = getFileIconInfo('', language)?.svg
+    const svgPath = icon?.path ?? mdiFileCodeOutline
+    const svgColor = icon?.color
```

Comment:
I think these need to be reactive:
```suggestion
    $: icon = getFileIconInfo('', language)?.svg
    $: svgPath = icon?.path ?? mdiFileCodeOutline
    $: svgColor = icon?.color
```